corehq/apps/hqwebapp/static/hqwebapp/less/_hq/includes/variables.less

```diff
@@ -23,17 +23,23 @@
 @cc-dark-warm-accent-mid: #ff8400;
 @cc-dark-warm-accent-low: #994f00;
 
-@cc-light-warm-accent-hi:   #f8ecbd;
-@cc-light-warm-accent-mid:  #f9c700;
-@cc-light-warm-accent-low:  #685300;
+@cc-light-warm-accent-extra-hi:   #fcf2cd;
```

Comment:
obv not actionable for this PR. but how is the feeling for the naming convention of `light-warm-accent` etc...not sure if this is clearer than color names?

Reply:
I think it makes sense for us to have a full palette of colors with designy names like `light-warm-accent` but also that devs writing application-level styles should usually be using more semantic names. So I've been thinking about leaving the `light-warm-accent`-type names but just adding more semantic synonyms to this file, like `@cc-link: @cc-brand-mid;` - what do you think?